jsyntrax/src/main/java/org/atpfivt/jsyntrax/generators/elements/Element.java

```diff
@@ -46,6 +46,13 @@ public void scale(double scale) {
         end.s = (int)(end.s * scale);
     }
 
+    public String getAnyTag() {
```

Comment:
Method is declared but not used and thus it should be deleted. Pay attention to IDE warnings!

Reply:
https://github.com/atp-mipt/jsyntrax/blob/add_title/jsyntrax/src/main/java/org/atpfivt/jsyntrax/generators/SVGCanvas.java#L74
Used here